modules/phase_field/include/action/ConservedAction.h

```diff
@@ -0,0 +1,54 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+#ifndef CONSERVEDACTION_H
+#define CONSERVEDACTION_H
+
+// MOOSE includes
+#include "Action.h"
+// MOOSE includes
+#include "Conversion.h"
+#include "FEProblem.h"
+#include "Factory.h"
+#include "MooseObjectAction.h"
+#include "MooseMesh.h"
+#include "AddVariableAction.h"
```

Comment:
You should move most of these includes into the .C files (use forward declarations if necessary). In the past months we worked very hard to streamline the includes in the headers to reduce compile times.